package.json

```diff
@@ -1,22 +1,43 @@
 {
   "name": "stellar-base",
   "version": "8.2.2",
-  "description": "Low level stellar support library",
+  "description": "Low-level support library for the Stellar network.",
   "main": "./lib/index.js",
   "types": "./types/index.d.ts",
+  "type": "module",
   "scripts": {
-    "test": "gulp test:node",
-    "test:watch": "gulp test:watch",
-    "docs": "jsdoc -c .jsdoc.json --verbose",
-    "dtslint": "dtslint types --localTs node_modules/typescript/lib",
-    "preversion": "gulp test",
-    "version": "gulp build",
-    "postversion": "git push && git push --tags",
-    "prettier-all": "prettier --write **/*.js"
+    "build": "yarn build:node",
```

Comment:
Should be this, no?
```suggestion
    "build": "yarn build:all",
```

Reply:
my intent was for `yarn build` to just do the node stuff for the local dev+test cycle because the browser stuff is slower, but I can see how that'd be misleading (under the mindset of "the browser stuff should 'just work' if I did the node stuff correctly, so I'd rather opt-in to build+testing it explicitly)

Reply:
ok, cleaned them up to be a little more consistent!